huaweicloud/services/acceptance/obs/resource_huaweicloud_obs_bucket_test.go

```diff
@@ -440,6 +463,38 @@ func testAccCheckObsBucketLogging(name, target, prefix string) resource.TestChec
 	}
 }
 
+func testAccCheckObsBucketStorageInfo(name string, size int64, objectNumber int) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[name]
+		if !ok {
+			return fmt.Errorf("not found: %s", name)
+		}
+
+		conf := acceptance.TestAccProvider.Meta().(*config.Config)
+		obsClient, err := conf.ObjectStorageClient(acceptance.HW_REGION_NAME)
+		if err != nil {
+			return fmt.Errorf("error creating OBS client: %s", err)
+		}
+
+		output, err := obsClient.GetBucketStorageInfo(rs.Primary.ID)
+		if err != nil {
+			return fmt.Errorf("error getting storageInfo of OBS bucket: %s", err)
+		}
+
+		if output.Size != size {
+			return fmt.Errorf("%s.storageInfo: Attribute 'size' expected %d, got %d",
+				name, output.Size, size)
+		}
+
+		if output.ObjectNumber != objectNumber {
+			return fmt.Errorf("%s.storageInfo: Attribute 'object_number' expected %d, got %d",
+				name, output.ObjectNumber, objectNumber)
+		}
+
+		return nil
+	}
+}
```

Comment:
This method is unnecessary. `resource.TestCheckResourceAttr(resourceName, "xxx")` just enough.

Reply:
done